app/components/Chart/GlobalEventsEvolution.vue

```
@@ -202,13 +202,102 @@ function getTrend({
+const landmarks = [
+  {
+    date: '2026-06-12', // FIXME: for prod, replace with 2026-07-01
+    name: 'Sample update',
+    description: '13 repositories added to the dataset',
+  },
+]
+
+const keyDates = computed(() => {
+  const dateList = dates.value ?? []
+  const lastDate = dateList.at(-1)
+  if (!lastDate) return []
```

ESLint

[error] Expected { after 'if' condition.

(curly)